test_runner/src/main/kotlin/ftl/shard/TestMethodDuration.kt

```diff
@@ -0,0 +1,35 @@
+package ftl.shard
+
+import ftl.args.AndroidArgs
+import ftl.args.IArgs
+import ftl.reports.xml.model.JUnitTestCase
+import ftl.reports.xml.model.JUnitTestResult
+
+fun createTestMethodDurationMap(junitResult: JUnitTestResult, args: IArgs): Map<String, Double> {
+    val junitMap = mutableMapOf<String, Double>()
+
+    // Create a map with information from previous junit run
+
+    junitResult.testsuites?.forEach { testsuite ->
```

Comment:
How about rewriting this to more functional style? Something like
```
return junitResult.testsuites?.asSequence()
        ?.mapNotNull { it.testcases }?.flatten()
        ?.filterNot { it.empty() || it.time == null }
        ?.mapNotNull { it.time?.run { (if (args is AndroidArgs) it.androidKey() else it.iosKey()) to it.time.toDouble() } }
        ?.filter { it.second >= 0 }
        ?.toMap() ?: emptyMap()
```

It's not change request, just thinking.

Reply:
No, no, no!!! xD

Reply:
> How about (...)

...or not :)

Reply:
so no :)